examples/63_Conic.html

```diff
@@ -36,12 +37,55 @@
 		              {name:"Co", type:"ConicBy5", args:["A","B","C","D","E"],color:[0,0,1],alpha:1,size:1},
 //		     {name:"l", type:"Join", args:["A", "D"]}
                       ];
-            CindyJS({canvasname:"CSCanvas",
+            var cdy=CindyJS({canvasname:"CSCanvas",
                         movescript:"csmove",
                         grid:1,
                         snap:true,
                         geometry:gslp});
 
+            var cases = [
+                ["parabola1", [0,0,1], [1,1,1], [2,4,1], [-2,4,1], [-1,1,1]],
+                ["parabola2", [0, 0, 1], [0, 2, 1], [6, 2, 1], [2, 6, 1], [2, 0, 1]],
+                ["hyperbola1", [2,2,1], [1,4,1], [4,1,1], [-4,-1,1], [-1,-4,1]],
+                ["hyperbola2", [-0.3838383838383838,1,-0.2525252525252525], [0.25,1,0.25], [1,0.02500000000000002,-0.12499999999999997], [1,0.6298342541436464,0.138121546961326], [1,-0.8571428571428571,0.14285714285714285]],
+                ["degen1", [0,4,1], [0,3,1], [0,-2,1], [5,0,1], [-6,0,1]],
+                ["degen2", [0,1,0.25], [0,1,0.3333333333333333], [0,1,-0.5], [1,0.23931623931623938,0.21367521367521372], [1,0,-0.16666666666666666]],
+                ["degen3", [0,1,0.25], [0.9999999999999999,-0.7826086956521738,-0.3623188405797101], [0,1,-0.5], [1,0,0.2], [1,0,-0.16666666666666666]],
+["degen4", [-0.25, 1, 0.125], [0.4953271028037384, 1, 0.23364485981308416], [1, 0.33333333333333326, 0.16666666666666663], [1, 1, 0.5], [0.5, 1, 0.5]],
+                ["degen5", [-0.25, 1, 0.125], [0.5643564356435643, 1, 0.24752475247524752], [1, 0.3333333333333333, 0.16666666666666666], [1, 1, 0.5], [0.5, 1, 0.5]],
+                ["degen6", [-0.4962406015037595, 1, 0.09398496240601504], [0.4, 1, 0.2], [1, 0.3333333333333333, 0.16666666666666666], [1, 0.6666666666666666, 0.3333333333333333], [1, 1, 0.5]],
+                ["degen7", [-0.09883720930232559, 1, 0.1453488372093023], [0.4, 1, 0.2], [1, 0.3333333333333333, 0.16666666666666666], [1, 0.6666666666666666, 0.3333333333333333], [1, 1, 0.5]],
+                ["degen8", [-0.5111111111111112, 1, 0.0925925925925926], [0.4, 1, 0.2], [1, 0.3333333333333333, 0.16666666666666666], [1, 0.6666666666666666, 0.3333333333333333], [1, 1, 0.5]],
+                ["almost degen", [0,1,0.25], [0.9999999999999999,0.16666666666666666,0.09689922480620154], [0,1,-0.5], [1,0.23931623931623938,0.21367521367521372], [1,0,-0.16666666666666666]],
+                ["almost2", [0,1,0.25], [1,0.16666666666666674,0.09689922480620156], [0.6105263157894737,1,-0.26315789473684215], [1,0.24545454545454548,0.2272727272727273], [1,0,-0.16666666666666666]],
+                ["almost3", [0,1,0.25], [1,0.7410071942446044,0.1798561151079137], [-0.8492063492063492,1,-0.1984126984126984], [1,0,0.2], [1,0,-0.16666666666666666]],
+                ["circle1", [6,1,1], [10,3,1], [10,9,1], [2,9,1], [2,3,1]],
```

Comment:
The `opps#` look to me like four collinear points and a fifth one elsewhere, so the conic would be ill defined in that case. The randomness we see is not related to the drawing step, but happens before.